commit 1fadac2354f700b293bccade044dc74be2ccf13a (refs/changes/86/886/3)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-14T10:40:57-08:00 (2 years, 11 months ago)
    
    MANTA-2953 minfo not present in marlin jobs
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/Makefile b/Makefile
index dce7b7f..b7e08e7 100644
--- a/Makefile
+++ b/Makefile
@@ -296,8 +296,8 @@ $(PROTO_BOOT_ROOT)/setup.sh:
 # define a target that re-invokes "make" again after node_modules has been
 # installed.
 #
-USER_TOOLS_MANTA = mfind mget mjob mln mlogin mls mmd5 mmkdir mput mrm mrmdir \
-    msign muntar
+USER_TOOLS_MANTA = mchattr mchmod mfind mget minfo mjob mln mlogin mls mmd5 \
+    mmkdir mput mrm mrmdir msign muntar
 USER_TOOLS_MCB = maggr mcat mpipe msplit mtee
 PROTO_USER_TOOLS_ROOT = $(PROTO_MARLIN_ROOT)/ubin
 
